utils/load/load.py

```diff
@@ -0,0 +1,46 @@
+"""
+This locust test randomly gets CIDs from a
+storetheindex daemon. CIDs are chosen randomly following a 
+ZIPF distribution. 
+"""
+
+import numpy as np
+
+import time
+from locust import HttpUser, task, between
+import random
+
+# Initializing Zipf distribution
+a = 2.  # Zipf parameter (feel free to change it)
+
+# Read from a list of imported cids
+def read_cids(filepath):
+    file1 = open(filepath, 'r')
```

Comment:
You need to close the file.
```suggestion
    with open(filepath) as file1:
```

Also, the 'r' is not needed as that is the default for opening.